### ISSUE TITLE
gzip-cli: no one pattern is specified. Operation is skipped.

### ISSUE THREAD

**Issue by gonzofish** [pakhuta/gzip-cli#8]
I've pulled the exported `gzip` function into my code:

```javascript
const gzip = require('gzip-cli').gzip;

gzip({
  outputExtensions: ['gz'],
  patterns: [
    './build/css/*.css',
    './build/js/*.js',
    './build/media/*.*',
  ]
});
```

This succeeds, but I also get the message:

```
gzip-cli: no one pattern is specified. Operation is skipped.
```

Looking at the code, it looks like when the library is `require`d, [it calls `run`](https://github.com/pakhuta/gzip-cli/blob/master/src/index.ts#L99) which would cause this message. 

What's confusing is that I'm calling it from a module, so I would assume that the [`module.parent` line](https://github.com/pakhuta/gzip-cli/blob/master/src/index.ts#L85) would be hit, but it doesn't seem like it is.

Would wrapping `run` in that check work?

```javascript
if (module.parent) {
  run();
}
```

~~EDIT: I think it should be `module.__parent__`, not `module.parent`~~

**Comment by pakhuta** [comment]
Sorry for long response. Github didn't notify me. I'm going to deal with it towards the end of the week.
Thanks for using `gzip-cli` ☺️

**Comment by gonzofish** [comment]
No worries! It still works fine just outputs that message. 

Thanks for the library 😄

**Comment by pakhuta** [comment]
Hi @gonzofish,
Good news: issue is fixed in version [1.1.1](https://github.com/pakhuta/gzip-cli/releases/tag/1.1.1).
I would appreciate it if you could check and provide your feedback.

**Comment by gonzofish** [comment]
Thanks @pakhuta I'll give it a try :)
